codice/alliance
# CAL-49 Add support for MTI data embedded in NITFs.

#### What does this PR do?
Adds the ability to produce a GMTI metacard from a NITF containing GMTI data.

#### Who is reviewing it?
@jlcsmith 
@kcwire 
@rzwiefel 
@lcrosenbu 

#### How should this be tested?
  install imaging-app
  ingest a NITF containing an image
  ingest a NITF containing only GMTI data
  verify the results on the search screen

#### Any background context you want to provide?
In cases where the NITF contains both image data and MTI data, it's ingested as an image.

#### What are the relevant tickets?
CAL-49

#### Screenshots (if appropriate)
N/A

#### Checklist:
